data/soapvalidator/MailClient/Calendar/Bugs/bug38478.xml

```diff
@@ -279,7 +279,7 @@
         </t:response>
     </t:test>
     
-      	<t:test depends="sendInviteReplyRequesta">
+      	<t:test>
```

Comment:
Why has this depends been removed?  If it is intentional, please update the commit comment so future people can understand the reason.  Same comment for line 434.

Reply:
There is no test with id 'sendInviteReplyRequesta' in the file so depends is not required.